homeassistant/components/wled/config_flow.py

```diff
@@ -0,0 +1,134 @@
+"""Config flow to configure the AdGuard Home integration."""
+import logging
+from typing import Any, Dict, Optional
+
+import voluptuous as vol
+from wled import WLED, WLEDConnectionError
+
+from homeassistant import config_entries
+from homeassistant.components.wled.const import DOMAIN
+from homeassistant.config_entries import SOURCE_ZEROCONF, ConfigFlow
+from homeassistant.const import CONF_HOST, CONF_MAC, CONF_NAME
+from homeassistant.helpers import ConfigType
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class WLEDFlowHandler(ConfigFlow):
```

Comment:
Pass the domain name as keyword argument to the class. We can remove the decorator.

https://developers.home-assistant.io/docs/en/config_entries_config_flow_handler.html#defining-your-config-flow